ui/page/components/account_selector.go

```diff
@@ -200,34 +204,35 @@ func (as *AccountSelector) Layout(gtx layout.Context) layout.Dimensions {
 }
 
 func (as *AccountSelector) ListenForTxNotifications(ctx context.Context) {
+	as.WL.MultiWallet.AddTxAndBlockNotificationListener(as.TxAndBlockNotificationListener, true, AccoutSelectorID)
```

Comment:
This might return an error. If a non-nil error is returned, log the error AND do not start the goroutine.